generators/spring-boot/generator.ts

```diff
@@ -461,7 +460,7 @@ public void set${javaBeanCase(propertyName)}(${propertyType} ${propertyName}) {
       },
       async generateKeyStore({ application }) {
         const keyStoreFile = this.destinationPath(`${application.srcMainResources}config/tls/keystore.p12`);
-        if (this.fakeKeytool) {
+        if (application.fakeKeytool) {
```

Comment:
fakeKeytool is a generator specific option.
it should not exist in application object.

Reply:
It's a command or/and genrator-application-data-object specific option ;-)

Reply:
It’s specific to the spring-boot generator.
And it’s not used in templates.

Actually is a test only feature.